src/components/Geogebra/GeogebraMergedAdminView.js

```diff
@@ -362,8 +362,11 @@ class GeogebraMergedAdminView {
             }
         }
 
+        const groupColor = this.getGroupColor(element.groupNum);
+
         this.evalXML(element.xml);
         this.evalCommand('UpdateConstruction()');
+        this.setDependentObjectsColor(element.name, groupColor)
```

Comment:
Months of working on js projects using linter made me hyper-aware of missing semicolons ;)
Please add them here and in other lines that don't have them.